enacts/wat_bal/maproom_monit.py

```diff
@@ -364,12 +374,13 @@ def wat_bal_plots(
         error_fig = pingrid.error_fig(error_msg="Grid box out of data domain")
         return error_fig
     precip.load()
+    taw = pingrid.sel_snap(xr.open_dataarray(Path(CONFIG["taw_file"])), lat, lng)
```

Comment:
I was not able to use taw here however defined earlier so had to read the file again. I am told that I reference taw before defining it.
The alignment is not necessary because it's only to fix precisions problems that are irrelevant to sel_snap here.
But it doesn't feel great to read file again.... does it?

Reply:
On the contrary, as a rule we *should* read the data from disk each time we need it.  Once there are more than a few maprooms, we won't be able to keep all our datasets in memory at all times.

It's ok to hold data in memory for the duration of a callback, but once the callback finishes we should release the memory to free it up for handling other callbacks.

Reply:
ok. Let's talk about this some day in the office. I don't think I get how the data requests flow.